libraries/standard/http/src/http_client.c

```diff
@@ -185,28 +200,31 @@ static HTTPStatus_t findHeaderInResponse( const uint8_t * pBuffer,
  * header field matched the header being searched for, and sets a flag to
  * represent reception of the header accordingly.
  *
- * @param[in] pHttpParser The parser object to which this callback is registered.
- * @param[in] pFieldLoc The location of the parsed header field in the response buffer.
+ * @param[in] pHttpParser The parsing context.
+ * @param[in] pFieldLoc The location of the parsed header field in the response
+ * buffer.
  * @param[in] fieldLen The length of the header field.
  *
- * @return Returns #HTTP_PARSER_CONTINUE_PARSING to indicate continuation with parsing.
+ * @return Returns #HTTP_PARSER_CONTINUE_PARSING to indicate continuation with
+ * parsing.
  */
 static int findHeaderFieldParserCallback( http_parser * pHttpParser,
                                           const char * pFieldLoc,
                                           size_t fieldLen );
 
 /**
  * @brief The "on_header_value" callback for the HTTP parser used by the
- * #findHeaderInResponse function. The callback sets the user-provided output parameters
- * for header value if the requested header's field was found in the
+ * #findHeaderInResponse function. The callback sets the user-provided output
+ * parameters for header value if the requested header's field was found in the
  * @ref findHeaderFieldParserCallback function.
  *
- * @param[in] pHttpParser The parser object to which this callback is registered.
- * @param[in] pVaLueLoc The location of the parsed header value in the response buffer.
+ * @param[in] pHttpParser The parsing context.
```

Comment:
This change is misleading as it can suggest that the parameter is of the `HTTPParsingContext_t` type. It is not a parsing context but the actual http parser object (of type `http_parser `) associated with the callback

Reply:
Same comment for callbacks where this doc change has been made

Reply:
The callback is not registered to this object, I will reword.